CRITICAL - 2021-11-24 00:08:47 --> Cannot declare class App\Controllers\Penulis, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-11-24 00:09:33 --> Cannot declare class App\Controllers\Penulis, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-11-24 00:09:34 --> Cannot declare class App\Controllers\Penulis, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-11-24 00:09:58 --> Undefined variable: penulis
#0 C:\Users\yerob\xampp\htdocs\ci4login\app\Views\admin\buku\listBuku.php(39): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\Users\\yerob\\...', 39, Array)
#1 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\View\View.php(214): include('C:\\Users\\yerob\\...')
#2 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('admin/buku/list...', Array, true)
#4 C:\Users\yerob\xampp\htdocs\ci4login\app\Controllers\Buku.php(20): view('admin/buku/list...', Array)
#5 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Buku->index()
#6 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Buku))
#7 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Users\yerob\xampp\htdocs\ci4login\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\yerob\\...')
#10 {main}
ERROR - 2021-11-24 03:16:33 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2021-11-24 03:16:33 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Users\yerob\xampp\htdocs\ci4login\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\BaseModel->first()
#5 C:\Users\yerob\xampp\htdocs\ci4login\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#6 C:\Users\yerob\xampp\htdocs\ci4login\vendor\myth\auth\src\Controllers\AuthController.php(88): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#7 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(824): Myth\Auth\Controllers\AuthController->attemptLogin()
#8 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#9 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\yerob\xampp\htdocs\ci4login\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\yerob\\...')
#12 {main}
CRITICAL - 2021-11-24 03:16:33 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\Users\yerob\xampp\htdocs\ci4login\app\Views\errors\html\error_exception.php:96
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\Users\\yerob\\...', 96, Array)
#1 C:\Users\yerob\xampp\htdocs\ci4login\app\Views\errors\html\error_exception.php(96): print_r(Object(Myth\Auth\Controllers\AuthController), true)
#2 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Debug\Exceptions.php(229): include('C:\\Users\\yerob\\...')
#3 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Debug\Exceptions.php(232): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Debug\Exceptions.php(116): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Obj
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-11-24 03:36:36 --> syntax error, unexpected end of file
#0 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('admin/buku/form...', Array, true)
#2 C:\Users\yerob\xampp\htdocs\ci4login\app\Controllers\Buku.php(33): view('admin/buku/form...', Array)
#3 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Buku->formTambah()
#4 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Buku))
#5 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Users\yerob\xampp\htdocs\ci4login\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\Users\yerob\xampp\htdocs\ci4login\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\yerob\\...')
#8 {main}
